support-frontend/assets/helpers/globalsAndSwitches/globals.test.ts

```diff
@@ -20,7 +20,7 @@ const getSpecifiedRegionAmountsFromGlobal = (
 	if (!testArray.length) {
 		return {};
 	}
-	return testArray[0];
+	return testArray[0] ?? {};
```

Comment:
Could move the check for undefined into the if statement above?

Reply:
Could do, it's a test which is why I did the inelegant solution